### PR TITLE
GCI : Task > PostGIS: Fix some strings in pgAdmin4 geometry viewer - …

### PR DESCRIPTION
**Pull request by ShivamRai2003**
PostGIS: Fix some strings in pgAdmin4 geometry viewer -Osgeo

### CODE REVIEW

File: web/pgadmin/static/js/sqleditor/geometry_viewer.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ let GeometryViewer = {
     if (self.panel_closed) {
       let wcDocker = window.wcDocker;
       let geometry_viewer_panel = handler.gridView.geometry_viewer =
-        handler.gridView.docker.addPanel('geometry_viewer',
+        handler.gridView.docker.addPanel('geometry_viewer'),
```
Comment (Gustry) [inline web/pgadmin/static/js/sqleditor/geometry_viewer.js]: This will break

Comment (ShivamRai2003) [reply web/pgadmin/static/js/sqleditor/geometry_viewer.js]: ok

Comment (ShivamRai2003) [reply web/pgadmin/static/js/sqleditor/geometry_viewer.js]: i will remove.

Comment (ShivamRai2003) [reply web/pgadmin/static/js/sqleditor/geometry_viewer.js]: Hi. I have remove now please review it

**Pull request closed without merging**